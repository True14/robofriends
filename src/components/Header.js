import React, { memo } from "react";
import CounterButton from "./CounterButton";

const Header = () => {
    return (
        <div>
            <h1>Robofriends</h1>
            <CounterButton color={"red"} />
        </div>
    )
}

export default memo(Header);