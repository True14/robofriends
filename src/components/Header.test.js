import React from "react";
import { shallow } from "enzyme";
import Header from "./Header";

describe("Test the Header component", () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(<Header />);
  });
  it("renders without crashing", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
