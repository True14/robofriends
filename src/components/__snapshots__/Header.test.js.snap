// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test the Header component renders without crashing 1`] = `
<div>
  <h1>
    Robofriends
  </h1>
  <Memo(Header)
    color="red"
  />
</div>
`;
